cats/src/io/github/iltotore/iron/cats.scala

```diff
@@ -127,7 +127,7 @@ object cats extends IronCatsInstances:
      * @return a [[Right]] containing this value as [[IronType]] or a [[Left]] containing the constraint message.
      * @see [[either]], [[eitherNel]].
      */
-    inline def eitherNec(value: A)(using inline c: Constraint[A, C]): EitherNec[String, T] = value.refineNec[C].map(_.asInstanceOf[T])
+    inline def eitherNec(value: A): EitherNec[String, T] = ops.either(value).toEitherNec
```

Comment:
@Iltotore I think we should stop the inlining for these methods now that they use `RuntimeConstraint`. It's no longer needed _plus_ primitives were anyways boxed because of the conversion to Either / Option / X / Y / Z monad